v22.1/ui-queues-dashboard.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Queues Dashboard
+summary: The Queues dashboard lets you monitor the health and performance of various queueing systems in CockroachDB.
+toc: true
+docs_area: reference.db_console
+---
+
+The **Queues** dashboard lets you monitor the health and performance of various queueing systems in CockroachDB.
+
+To view this dashboard, [access the DB Console](ui-overview.html#db-console-access) and click **Metrics** on the left-hand navigation, and then select **Dashboard** > **Queues**.
+
+{% include {{ page.version.version }}/ui/ui-metrics-navigation.md %}
+
+{{site.data.alerts.callout_info}}
+All timestamps in the DB Console are shown in Coordinated Universal Time (UTC).
+{{site.data.alerts.end}}
+
+The **Queues** dashboard displays the following time series graphs:
+
+## Queue Processing Failures
+
+<img src="{{ 'images/v22.1/ui_queue_failures.png' | relative_url }}" alt="DB Console queue failures graph" style="border:1px solid #eee;max-width:100%" />
+
+The **Queue Processing Failures** graph shows you processing failures experienced across various queuing systems.
+
+On hovering over the graph, the values for the following metrics are displayed:
+
+Metric | Description
```

Comment:
To increase readability, you might split the table into three columns and have the full metric name (metric namespace path?) as its own field, something like "Name | Full Metric(?) | Description"